components/tekton/tasks/get-kubeconfig.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-kubeconfig
+spec:
+  description: >-
+    This task can be used to gather kubeconfig file in the output workspace.
+
+    Pre-requirements: This task assumes 1) the hub cluster includes RHACM and
+    2) the name of the workload cluster is the same as the namespace where the
+    kubeconfig lives, and 3) the name of the workload cluster is the same as
+    the name of the kubeconfig file.
+  params:
+    - name: cluster-name
+      default: ""
+      description: |
+        Name of cluster to extract kubeconfig.
+        Namespace where the kubeconfig secret lives in the hub cluster.
+        Name of the kubeconfig file.
+      type: string
+  workspaces:
+    - name: kubeconfigs
+      description: |
+        Output directory for the kubeconfig output file.
+        By default it will use '/workspace/kubeconfigs'
+  steps:
+    - name: get-kubeconfig
+      image: quay.io/openshift/origin-cli:latest
```

Comment:
Could we try/utilize registry.redhat.io/openshift4/ose-cli:latest instead of the upstream image?

Reply:
Done.